### DISCUSSION TITLE
Scripts directory

### DISCUSSION THREAD

**Discussion by driftSam** [comictagger/comictagger#489] [General]
Hey,

I see in the README for the Script directory it say "This feature is UNSUPPORTED, and is for the convenience of development-minded
users of ComicTagger."

Does that mean that these scripts won't be updated to reflect changes in the project? 
It looks like some of the scripts were updated when the `ComicArchive` class was moved but the scripts in the main branch don't seem to work when run with version 1.5.5

Also, do question like this belong here or is the forum a better place for them?

**Comment by lordwelch** [comment]
Those scripts have not worked since before 1.1.30. They may just be fully removed in the future. Yes, you are in the right place

**Comment by driftSam** [comment]
Sounds good. Thanks for the info
